src/main/kotlin/com/exactpro/th2/ws/client/api/impl/WebSocketClient.kt

```diff
@@ -206,11 +206,12 @@ class WebSocketClient(
             try {
                 textFrames.clear()
                 binaryFrames.clear()
+                val connectUri: URIBuilder = URIBuilder(uri.toString())
```

Comment:
Maybe incapsulate builder into settings? This way we can avoid creating a builder for URI if `addQueryParam` method wasn't called. Also we should print actual URI in catch block below

Reply:
Done